test/views/behaviours/QuestionViewBehaviours.scala

```diff
@@ -71,7 +71,7 @@ trait QuestionViewBehaviours[A] extends ViewBehaviours {
           "show an error summary" in {
 
             val doc = asDocument(createView(form.withError(FormError(field, "error"))))
-            assertRenderedById(doc, "error-summary-title")
+            assertRenderedByClass(doc, "error-summary-title")
```

Comment:
Change "error-summary-title" to "govuk-error-summary"

Reply:
Done